config/locales/views/documents.en.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  documents:
+    document_name: File name 
+    download_file: Download
```

Comment:
ref the the previous `.key.lookup` comment before ;
`download_file` sould probably be under the `index` key